app/views/questions/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<!-- <p id="notice"><%= notice %></p> -->
+
+<% if can? :create, Question %>
+  <div class="row">
+    <%= link_to 'Ask Question', new_question_path, class: "btn btn-primary right" %>
+  </div>
+  <br>
+<% end %>
+
+<div class="row">
+  <h4 class="sorted_area left">
+    Sorted by: <strong><%= @value %></strong>
+  </h4>
+  <div class="sort_by dropdown right">
+    <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Sort by <span class="caret"></span></button>
+    <ul class="dropdown-menu">
+      <li><%= link_to " Newest Questions", questions_path(value: "Newest Questions") %></li>
+      <li><%= link_to " Oldest Questions", questions_path(value: "Oldest Questions") %></li>
+      <li><%= link_to " Title", questions_path(value: "Question Titles") %></li>
+      <li><%= link_to " Max Answers", questions_path(value: "Max Answers") %></li>
+      <li><%= link_to " Min Answers", questions_path(value: "Min Answers") %></li>
+    </ul>
+  </div>
+</div>
+
+<div class="container_body">
+  <% @questions.each do |q| %>
```

Comment:
`q` doesn't qualify as a good variable name. A general rule is that when you are looping over a collection then the block's parameter should be singular of that collection name. so here `q` should be replaced with `question`